clouddrift/ragged.py

```diff
@@ -667,7 +668,7 @@ def subset(
     :func:`apply_ragged`
     """
     mask_traj = xr.DataArray(
```

Comment:
Should we replace in the code the `mask_traj` variables by mask_row`? It is not needed bu t will be consistent.